index.html

```diff
@@ -3064,41 +3099,30 @@ <h3>Verify Proof</h3>
 the `proof` value removed.
           </li>
           <li>
-Let <var>transformedData</var> be the result of
-<a href="#dfn-transformation">transforming</a> the <var>unsecuredDocument</var>
-according to a <a>transformation algorithm</a> associated with the
-<a>cryptographic suite</a> specified in <var>proof</var> and the
-<var>options</var> parameters provided as inputs to the algorithm.
-The type of <a>cryptographic suite</a> is specified by the
-<var>proof.type</var> value and MAY be further described by
-<a>cryptographic suite</a>-specific properties expressed in <var>proof</var>.
+Let <var>transformedData</var> be the result of running |suite|'s'
+[=cryptographic suite/transform=] algorithm on the <var>unsecuredDocument</var>.
         </li>
           <li>
-Let <var>hashData</var> be the result of
-<a href="#dfn-hashing">hashing</a> the <var>transformedData</var>
-according to a <a>hashing algorithm</a> associated with the
-<a>cryptographic suite</a> specified in the <var>proof</var> and
-<var>options</var> parameters provided as inputs to the algorithm.
+Let <var>hashData</var> be the result of running |suite|'s'
+[=cryptographic suite/hash=] algorithm on the <var>transformedData</var>.
           </li>
           <li>
-Let <var>isProofVerified</var> be the result of running the
-<a>proof verification algorithm</a> associated with the
-<a>cryptographic suite</a> with the <var>hashData</var> and <var>options</var>
-parameters provided as inputs to the algorithm.
+Let |isProofVerified:valid or invalid| be the result of running |suite|'s
```

Comment:
```suggestion
Let |isProofVerified:valid| or |isProofVerified:invalid| be the result of running |suite|'s
```

Reply:
Ah, crap... I merged w/o seeing these... raise another PR to fix these, please, @TallTed.... note that I'm getting ready to merge the other PR from @jyasskin in a bit, so maybe wait a day for this stuff to settle before doing your PR.

Reply:
@TallTed's suggestion isn't the right fix here, although I agree that what I wound up with is awkward. This is the `|variable:type|` syntax from https://respec.org/docs/#giving-variables-data-types, and the type of the variable is actually the enum type of "valid" or "invalid". Respec doesn't accept `"` inside its |var| syntax, and it doesn't appear to have another way of attaching types, so I dropped the quotes. One could argue that I should have instead given up on assigning a type to this variable.

Reply:
@jyasskin — I think you're in a better position than I to log a ReSpec issue, about how to type such enum-typed vars ... whether that leads to some complex manual process, an improvement of ReSpec, or otherwise.

Reply:
Indeed, I am. And in figuring out what issue to file, I found the markup I should have used: https://github.com/w3c/respec-web-services/pull/399 and https://github.com/w3c/respec/wiki/data-type.

Reply:
@jyasskin — As you've noted that _Respec doesn't accept `"` inside its `|var|` syntax_, it seems that some addition detail should be in [that wiki](https://github.com/w3c/respec/wiki/data-type) (and hence the proper ReSpec documentation) to cover such scenarios as this enum typing.

Reply:
Sure, done: https://github.com/w3c/respec/wiki/data-type. Along with the bug to make it support quotes.